geocube/geo_utils/geobox.py

```diff
@@ -37,19 +39,18 @@ def geobox_from_rio(xds):
 
     Returns
     -------
-    :obj:`datacube.utils.geometry.GeoBox`
+    :obj:`odc.geo.geobox.GeoBox`
 
     """
     height, width = xds.rio.shape
     try:
         transform = xds.rio.transform()
     except AttributeError:
         transform = xds[xds.rio.vars[0]].rio.transform()
-    return geometry.GeoBox(
-        width=width,
-        height=height,
+    return GeoBox(
+        shape=XY(x=width, y=height),
         affine=transform,
-        crs=geometry.CRS(crs_to_wkt(xds.rio.crs)),
+        crs=CRS(crs_to_wkt(xds.rio.crs)),
```

Comment:
This should be able to accept `xds.rio.crs` directly, so unless you want to be explicit and force wkt representation there is not need to wrap it in CRS